### PR TITLE
Task-203590 IP-verification flow for enrollment verification

### PR DESCRIPTION
**Pull request by GitVivekHub**
# I have ensured that following `Pull Request Checklist` is taken care of before sending this PR

* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [x] Updated test cases in test-cases-tracker
* [x] Updated new API endpoint if any in common postman collection
* [x] Updated DB changes in db-tracker if any

### CODE REVIEW

**Review by parthgandhitekdi — APPROVED** [review]

File: src/src/beneficiaries/beneficiaries.service.ts [hunk 1]
```diff
@@ -2577,6 +2703,7 @@ export class BeneficiariesService {
 							user_id: req.id,
 							status,
 							reason_for_status_update: reason,
+							enrollment_verification_status:updatedUser.program_beneficiaries?.enrollment_verification_status === "change_required"?"reverification_required":"pending"
```
Comment (manojLondhe) [inline src/src/beneficiaries/beneficiaries.service.ts:2706]: add spaces before and after `:` and `?`

File: src/src/beneficiaries/beneficiaries.controller.ts [hunk 2]
```diff
@@ -287,4 +287,35 @@ export class BeneficiariesController {
 			response,
 		);
 	}
+
+	@Post('verify-enrollment')
```
Comment (manojLondhe) [inline src/src/beneficiaries/beneficiaries.controller.ts:291]: change as admin/verify-enrollment

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: src/src/beneficiaries/beneficiaries.service.ts (hunk 1 above)
Comment (GitVivekHub) [reply src/src/beneficiaries/beneficiaries.service.ts]: comment resolved

File: src/src/beneficiaries/beneficiaries.controller.ts (hunk 2 above)
Comment (GitVivekHub) [reply src/src/beneficiaries/beneficiaries.controller.ts]: comment resolved

**Pull request merged**